packages/core/src/client.ts

```diff
@@ -3830,6 +3830,10 @@ export class MedplumClient extends TypedEventTarget<MedplumClientEventMap> {
           const newState = (e.newValue ? JSON.parse(e.newValue) : undefined) as LoginState | undefined;
           if (oldState?.profile.reference !== newState?.profile.reference) {
             window.location.reload();
+          } else if (newState) {
+            this.setAccessToken(newState.accessToken, newState.refreshToken);
+          } else {
+            this.clear();
```

Comment:
I think the  case where `newState` and `oldState` are both `null` is where this `else` clause would get hit.
But that seems extremely unlikely since it basically means you deleted `activeLogin` key when it was already empty, in which case `this.clear()` is probably a no-op.

I think we may be able to remove this case since none of our tests are hitting it and the only test we could add is one that is probably not a real case that would come up ever